### ISSUE TITLE
Set log handlers as internal instead of private

### ISSUE THREAD

**Issue by fastackl** [Vectorized/dn404#138]
<img width="807" alt="image" src="https://github.com/Vectorized/dn404/assets/80379118/df3606ef-1331-4a69-a555-14ba3bf6c7d1">

These helpers are very useful and would be great to be able to inherit instead of rewriting. That contract size limit can sneak up on you.

Unless there's a reason these are set to private instead of internal 🤔

**Comment by Vectorized** [comment]
These stuff use some _extremely_ dangerous memory tricks tho. It’s like handling super sharp knifes.

**Comment by fastackl** [comment]
Yup I agree. My use case is I want to fire off mirror transfer events in a derived contract. I don't need to modify the packed log handlers, just access them.

But if I can't inherit these beautiful handlers then the only way to fire these events (other than rewrite something myself) is to copy paste the packed log handlers in my derived contract.

I.e. setting as private doesn't stop devs from handling knives anyway just forces devs to duplicate code 😅
